cloud/aws/variables.tf

```diff
@@ -6,6 +6,11 @@ variable "cockroach_port" {
   default = "26257"
 }
 
+# Comma-separated list of directories added as auxiliary stores.
+variable "aux_data_dirs" {
+  default = ""
```

Comment:
ok. I'll rename the variable to `stores` and make sure we don't hard-code `data` anywhere.